### DISCUSSION TITLE
Log doesn't have date

### DISCUSSION THREAD

**Discussion by gqqnbig** [seaweedfs/seaweedfs#3123] [Q&A] [answered]
I use code like this to start weed services

`nohup ./weed volume -max=0 -mserver="aloha:9333" -dir="/data/brick1/sw" > volume.out 2>&1 &`

The log is like this
```
I0529 13:48:42 63349 file_util.go:23] Folder /data/brick1/sw Permission: -rwxr-xr-x
I0529 13:48:42 63349 disk_location.go:209] Store started on dir: /data/brick1/sw with 0 volumes max 0
I0529 13:48:42 63349 disk_location.go:212] Store started on dir: /data/brick1/sw with 0 ec shards
```

The log contains time but no date.

How can date be added to the log?

**Comment by chrislusf** [comment, marked as answer]
0529 is the date

**Reply by gqqnbig** [reply]
Oh my god.. I thought it's like error code or info code.. Thank you.
